test/TestBuilder.js

```diff
@@ -67,6 +69,8 @@ function resolveTestLibsPlugin() {
 					return path.resolve("../node_modules/mithril/test-utils/browserMock.js")
 				case "ospec":
 					return ("../node_modules/ospec/ospec.js")
+				case "testdouble":
```

Comment:
it is but then it breaks, probably because of:

> // We use this homebrew plugin so that libs are copies to _virtual folder and *not* build/node_modules
// (which would be the case with preserve_modules).
// Files in build/node_modules are treated as separate libraries and ES mode resets back to commonjs.

Reply:
having a case with fallthrough also works, so we removed it